src/makeCalculator.js

```diff
@@ -37,7 +37,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  // object created
+  const calculator = {
+    result: 0,
+
+    // created callback operator methods
+    add(prevValue, number) {
+      return prevValue + number;
+    },
```

Comment:
why one number is prev value? and other one is simple number?
let's think more about variable names, they should have the same meaning